x/wasm/internal/keeper/cosmwasm/query.go

```diff
@@ -0,0 +1,18 @@
+package cosmwasm
+
+type IBCQuery struct {
```

Comment:
Yeah,  didn't even think about the queries. 

This is a good point, we should define them well.

Reply:
Not sure if this is a useful scenario but I thought a contract may want to query all it's channels and broadcast to it's counter parties. Or may close them all (end-of-life scenario)